### DISCUSSION TITLE
Stable Version Confusion

### DISCUSSION THREAD

**Discussion by PeterKW** [debauchee/barrier#1275] [Q&A]
winget show barrier
Found Barrier [DebaucheeOpenSourceGroup.Barrier]
Version: 2.3.3-release
Publisher: Debauchee Open Source Group
Description: Barrier is open-source KVM software
Homepage: https://github.com/debauchee/barrier/
License: GNU General Public License v2.0
Installer:
  Type: Inno
  Download Url: https://github.com/debauchee/barrier/releases/download/v2.3.3/BarrierSetup-2.3.3-release.exe
  SHA256: 13090fda8e71b3fd1e21c7a1f495d28731b18faa12b3d7d1a0ff1bedd0ab5257

Sorry, slightly confused from the documentation what version I should run? winget seems to be installing a release from more than a year ago, is this the most secure/stable version to run at the moment? Not wanting to run developer versions on this device at the moment.

**Comment by shymega** [comment]
Yes, that's the latest version.

On Mon, Aug 2, 2021, at 3:38 PM, Peter wrote:
> 
> 
> winget show barrier
> Found Barrier [DebaucheeOpenSourceGroup.Barrier]
> Version: 2.3.3-release
> Publisher: Debauchee Open Source Group
> Description: Barrier is open-source KVM software
> Homepage: https://github.com/debauchee/barrier/
> License: GNU General Public License v2.0
> Installer:
> Type: Inno
> Download Url: 
> https://github.com/debauchee/barrier/releases/download/v2.3.3/BarrierSetup-2.3.3-release.exe
> SHA256: 13090fda8e71b3fd1e21c7a1f495d28731b18faa12b3d7d1a0ff1bedd0ab5257
> 
> Sorry, slightly confused from the documentation what version I should 
> run? winget seems to be installing a release from more than a year ago, 
> is this the most secure/stable version to run at the moment? Not 
> wanting to run developer versions on this device at the moment.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/debauchee/barrier/issues/1238>, or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AAKF2QFFXRKJP3DC4UI6TJ3T22UX5ANCNFSM5BM4CGOA>.
> 


-- 
Kind regards,
--
Dom Rodriguez (also known as shymega)
